grommet/grommet
# Hotfix wrong override when value is not defined in breakpoint

#### What does this PR do?

This PR tend to avoid generating a wrong CSS override when value is not known in theme breakpoint (Same as for `gap`, etc...)

#### Where should the reviewer start?

Read the linked issue to see the difference between both Codepens.

#### What testing has been done on this PR?

Manual tests only

#### How should this be manually tested?

Use Developper console with the new version to figure out that CSS override have not been generated for unknown value.
Then add a 'small' breakpoint with distinct edgeSize value to see the override being generated.

#### Do Jest tests follow these best practices?

No Jest tests.

#### Any background context you want to provide?

#### What are the relevant issues?

Issue #6968 

#### Screenshots (if appropriate)

N/A

#### Do the grommet docs need to be updated?

No

#### Should this PR be mentioned in the release notes?

Maybe, depending if a lot of people get a specific expected behavior with this bug.

#### Is this change backwards compatible or is it a breaking change?

It should be backwards compatible, except if some users get a specific behavior getting the raw CSS value inserted and being different than the value defined in global theme.
